org.eclipse.scout.rt.uinotification/src/main/java/org/eclipse/scout/rt/api/uinotification/UiNotificationRegistry.java

```diff
@@ -372,6 +377,21 @@ public void removeListener(String topic, UiNotificationListener listener) {
     }
   }
 
+  /**
+   * Retrieves how many observer are listening for the topic given.
```

Comment:
observer -> observers
topic given -> given topic
 for given topic. -> for the given topic.